File: app/src/main/java/vn/loitp/app/activity/demo/ebookwithrealm/adapters/RealmModelAdapter.java
package vn.loitp.app.activity.demo.ebookwithrealm.adapters;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;

import io.realm.RealmBaseAdapter;
import io.realm.RealmObject;
import io.realm.RealmResults;

public class RealmModelAdapter<T extends RealmObject> extends RealmBaseAdapter<T> {

    public RealmModelAdapter(Context context, RealmResults<T> realmResults, boolean automaticUpdate) {

        super(context, realmResults, automaticUpdate);
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent) {

        return null;
    }
}

File: app/src/main/java/vn/loitp/app/a/demo/ebookWithRealm/adapters/RealmModelAdapter.kt
package vn.loitp.app.a.demo.ebookWithRealm.adapters

import android.content.Context
import android.view.View
import android.view.ViewGroup
import io.realm.RealmBaseAdapter
import io.realm.RealmObject
import io.realm.RealmResults

open class RealmModelAdapter<T : RealmObject> internal constructor(
    context: Context,
    realmResults: RealmResults<T>,
    automaticUpdate: Boolean
) : RealmBaseAdapter<T>(
    context,
    realmResults,
    automaticUpdate
) {

    override fun getView(position: Int, convertView: View, parent: ViewGroup): View? {
        return null
    }
}
